source/nodejs/adaptivecards-controls/src/utils.ts

```diff
@@ -138,3 +138,12 @@ export function getAttributeValueAsInt(element: HTMLElement, attributeName: stri
 
     return defaultValue;
 }
+
+const ttPolicy = window.trustedTypes?.createPolicy('adaptivecards-controls', {
+    createHTML: value => value,
+});
+
+export function setInnerHtml(element: HTMLElement, htmlContent: string) : void {
+    const trustedHtml = ttPolicy?.createHTML(htmlContent) ?? htmlContent;
+    element.innerHTML = trustedHtml as string;
+}
```

Comment:
with the `emptyHTML` property I mention below, we shouldn't need to define/create a policy -- should only need the `clearElement` helper

Reply:
the policy is shared, and we have to have policy for this

![image](https://user-images.githubusercontent.com/6290692/189410681-924f6e5b-08d4-4e54-8d20-623f2c6a9f71.png)

I can create another clearElement if you really want to go that way.

Reply:
fixed